vehicles/src/modules/special-auth/entities/no-fee-type.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { Entity, Column, PrimaryColumn } from 'typeorm';
+import { AutoMap } from '@automapper/classes';
+import { Base } from 'src/modules/common/entities/base.entity';
+
+@Entity({ name: 'permit.ORBC_NO_FEE_TYPE' })
+export class NoFeeType extends Base {
```

Comment:
I dont think you can extend base as it has application metadata audit fields

Reply:
dto is gone, so we are okay.

Reply:
NoFeeType is a type table. It does not have metadata field defined in base entity such as APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_CREATE_USER_GUID, APP_CREATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_GUID, APP_LAST_UPDATE_TIMESTAMP.

Please do not extend base entity

Reply:
removed entity file, not needed anymore